packages/frontend/src/components/Explorer/ExploreTree/TableTree/Tree/TreeSingleNode.tsx

```diff
@@ -185,11 +198,11 @@ const TreeSingleNode: FC<Props> = memo(({ node }) => {
                         shadow="subtle"
                         withinPortal
                         withArrow
-                        disabled={!description && !isMissing}
+                        disabled={isHoverCardDisabled}
                         position="right"
                         radius="md"
                         /** Ensures the hover card does not overlap with the right-hand menu. */
-                        offset={isFiltered ? 80 : 40}
+                        offset={80}
```

Comment:
always offset to see it not overlapping the other filter + "..." icons

<img width="997" alt="image" src="https://github.com/user-attachments/assets/b100e52d-eaa2-4187-a6bf-d992d9894f94" />